skore/tests/unit/sklearn/plot/test_precision_recall_curve.py

```diff
@@ -1,24 +1,27 @@
 import matplotlib as mpl
 import numpy as np
 import pytest
+from sklearn.base import clone
 from sklearn.datasets import make_classification
 from sklearn.linear_model import LogisticRegression
 from sklearn.model_selection import train_test_split
-from skore import CrossValidationReport, EstimatorReport
+from skore import ComparisonReport, CrossValidationReport, EstimatorReport
 from skore.sklearn._plot import PrecisionRecallCurveDisplay
 from skore.sklearn._plot.utils import sample_mpl_colormap
 
 
 @pytest.fixture
 def binary_classification_data():
-    X, y = make_classification(random_state=42)
+    X, y = make_classification(class_sep=0.1, random_state=42)
```

Comment:
Modified the generation to make sure that if we don't compute the mean properly, then we can detect it. With this change, we would have detected the regression.